src/kmsg_tailer.c

```diff
@@ -0,0 +1,23 @@
+#include <err.h>
```

Comment:
Feels like there's enough common code between `src/kmsg_tailer.c` and `src/log_tailer.c` that we ought to be able to consolidate the two by adding a conditional for `type`, no?  Would help clear up parts of the elixir code too (`port_binary_name` wouldn't be necessary, for example).

Reply:
That's a good point. My goal was to make them separate processes so they I didn't have to deal with a protocol to distinguish between the different message types both going out `STDOUT`, but I didn't think about making a single binary that can run in both modes.